package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Paratii Smart Contracts",
   "main": "dist/paratii.js",
   "browser": {
-    "./lib/paratii.ipfs.uploader.js": "./lib/paratii.ipfs.uploader.browser.js"
+    "./lib/paratii.ipfs.uploader.js": "./lib/paratii.ipfs.uploader.browser.js",
+    "./dist/paratii.ipfs.uploader.js": "./dist/paratii.ipfs.uploader.browser.js"
```

Comment:
This is really the only change that is necessary to review

Reply:
@ya7ya ?

Reply:
Looks good to me 👍